drivers/clock_control/Kconfig.stm32wl

```diff
@@ -0,0 +1,65 @@
+# STM32WL PLL configuration options
+
+# Copyright (c) 2019 Linaro
```

Comment:
No copyright for ST?

Reply:
I removed this file, "Kconfig.stm32wl", as content was already merged into Kconfig.stm32l4_l5_wb_wl